roles/sap_ha_install_hana_hsr/tasks/hdbuserstore.yml

```diff
@@ -1,30 +1,20 @@
 ---
-
-# [A] Create and Store Connection Info in hdbuserstore for SYSTEMDB
-- name: SAP HSR - [A] Check if hdbuserstore exists
-  tags: hsr_hdbuserstore
-  become: true
+- name: "SAP HSR - Check if hdbuserstore exists"
   become_user: "{{ sap_ha_install_hana_hsr_sid | lower }}adm"
   shell: |
-        /usr/sap/{{ sap_ha_install_hana_hsr_sid}}/SYS/exe/hdb/hdbuserstore \
-        List {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }} 
+    /usr/sap/{{ sap_ha_install_hana_hsr_sid}}/SYS/exe/hdb/hdbuserstore \
+    List {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }}
   args:
     executable: /bin/bash
   register: sap_ha_install_hana_hsr_hdbuserstore
+  failed_when: false
+  changed_when: false
 
-- name: SAP HSR - [A] Skipping hsr_hdbuserstore
-  tags: hsr_hdbuserstore
-  debug:
-    msg: "Skipping creating {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }} hdbuserstore "
-  when: sap_ha_install_hana_hsr_hdbuserstore.rc==0
-
-- name: SAP HSR - [A] Create and Store Connection Info in hdbuserstore
-  tags: hsr_hdbuserstore
-  become: true
+- name: "SAP HSR - Create and Store Connection Info in hdbuserstore"
   become_user: "{{ sap_ha_install_hana_hsr_sid | lower }}adm"
```

Comment:
add become: true

Reply:
become: true is already set on playbook level and required for all other tasks to run. Just the user changes for a few tasks.

Reply:
@kksat I am now resolving this conversation after @ja9fuchs replied and explained it. I also see that you approved the PR already. Feel free to open an issue related to your comment.